modular_skyrat/modules/synths/code/surgery/mechanic_steps.dm

```diff
@@ -23,23 +28,29 @@
 	name = "pry off plating"
 	implements = list(
 		TOOL_CROWBAR = 100,
-		TOOL_HEMOSTAT = 10)
+		TOOL_HEMOSTAT = 10,
+	)
 	time = 24
 
 /datum/surgery_step/pry_off_plating/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	do_sparks(rand(5, 9), FALSE, target.loc)
 	return TRUE
 
 /datum/surgery_step/pry_off_plating/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	display_results(user, target, span_notice("You begin to pry off [target]'s [parse_zone(target_zone)] plating..."),
-			"[user] begins to pry off [target]'s [parse_zone(target_zone)] plating.",
-			"[user] begins to pry off [target]'s [parse_zone(target_zone)] plating.")
+	display_results(
+		user,
+		target,
+		span_notice("You begin to pry off [target]'s [parse_zone(target_zone)] plating..."),
+		"[user] begins to pry off [target]'s [parse_zone(target_zone)] plating.",
+		"[user] begins to pry off [target]'s [parse_zone(target_zone)] plating.",
+	)
 
 //weld plating
 /datum/surgery_step/weld_plating
 	name = "weld plating"
 	implements = list(
-		TOOL_WELDER = 100)
+		TOOL_WELDER = 100,
+	)
 	time = 24
```

Comment:
Honestly they should probably just be changed to `2.5` just so it's a coherent number, rather than a seemingly random one.
```suggestion
	time = 2.4 SECONDS
```